helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml

```diff
@@ -275,13 +275,6 @@ spec:
       record: aggregation:prometheus:memory_percentage
     - expr: sum(label_replace(container_memory_working_set_bytes{container='prometheus', namespace=~'.*-prometheus'}, "cluster_id", "$2", "pod", "(prometheus-)(.+)(-.+)")) by (cluster_type , cluster_id)
       record: aggregation:prometheus:memory_usage
-  - name: managed-apps.grafana-cloud.recording
```

Comment:
Remove this because the raw metric does not exist and the dashboard is currently not working https://giantswarm.grafana.net/d/zb9aCfNGk/managed-apps-error-budgets?orgId=1&from=now-4y&to=now-3y